### DISCUSSION TITLE
2 minor things for NFT enjoyers

### DISCUSSION THREAD

**Discussion by JJAYRAMB0X** [jdogresorg/freewallet-desktop#139] [Show and tell]
Hello ! 
Just wanted to mention 2 small things  and either get a discussion going or a hard no lol   either way.

1) Windows icon- not sure if there's something wrong with my version or what but this icon cannot stand: 
![image](https://github.com/jdogresorg/freewallet-desktop/assets/138930376/b20d6436-fa84-4900-bf20-c683660fc76d)
Hopefully that's viewable, I'm not sure if it's like that just for me or if that's the actual updated icon.. if it is, my suggestion would be 'anything but that'.

2) 'Gallery Tab':  Up at the top: 
![image](https://github.com/jdogresorg/freewallet-desktop/assets/138930376/2709a2fb-79b4-4dc2-b79d-76480d9eac1c)
There would be an awesome place to fit a 'Gallery' button where advanced features could be looked at later but initially, the ability to view all your NFTs at once in a nice layout, possibly 3 diff size choices, maybe launch a slideshow etc.

That's it for now! 
Thank you

**Comment by davestaxcp** [comment]
I agree with both of these ideas! Would be awesome.
